app/src/main/res/layout/loading_indicator.xml

```diff
@@ -7,6 +7,15 @@
     android:gravity="center"
     android:orientation="vertical">
 
+    <ImageView
+        android:src="@drawable/logotipas_nma_full"
+        android:layout_width="fill_parent"
```

Comment:
Use `match_parent` instead of `fill_parent`. From Google docs https://developer.android.com/reference/android/view/ViewGroup.LayoutParams.html
> FILL_PARENT (renamed MATCH_PARENT in API Level 8 and higher), which means that the view wants to be as big as its parent (minus padding)

> fill_parent: The view should be as big as its parent (minus padding). This constant is deprecated starting from API Level 8 and is replaced by match_parent.